src/themes/dark.ts

```diff
@@ -0,0 +1,15 @@
+import { AppTheme } from '.';
+
+export default {
+	backgroundColor: '#171717',
+	borderColorSelected: '#333333',
+	boxShadow:
+		'0px 0.9474522471427917px 7.579617977142334px 1.8949044942855835px #142E6E1A',
```

Comment:
Odd to see fractional pixels here; seems like previous values were not fractional pre-theming. Could you confirm these values are expected?

Reply:
I'll have to check this with the design team. I copied the values from Figma. Nice catch 🔍

Reply:
Just checked with them. They're OK with that and the only change they asked for was to make the anchor be colored gray (`#757575`).
This is the commit: 8e30c4415876cf697677ea0e9502136daa456245